clients/vscode/package.json

```diff
@@ -399,7 +399,7 @@
         "command": "tabby.chatView.focus",
         "key": "ctrl+l",
         "mac": "cmd+l",
-        "when": "!editorHasSelection && tabby.chatEnabled && !tabby.chatViewVisible"
+        "when": "!editorHasSelection && editorTextFocus && tabby.chatEnabled"
```

Comment:
with focus works as expected, chatView.focus shall be used for cmd-l when chatView is visible as well